DESCRIPTION

```diff
@@ -16,7 +16,7 @@ Authors@R: c(person(given = "Laurent", family = "Gatto",
                     role = c("aut", "cre"),
 		            comment = c(ORCID = "0000-0001-7406-4443")),
              person(given = "Sigurdur", family = "Smarason", role = "ctb"),
-	           person(given = "Thomas", family = "Naake", role = "ctb")
+	         person(given = "Thomas", family = "Naake", role = "ctb")
```

Comment:
Could you please fix this indentation (I thought I already did that, maybe just (re)merge with master).
```suggestion
	           person(given = "Thomas", family = "Naake", role = "ctb")
```